### PR TITLE
Beta testing list: images should fit in container

### PR DESCRIPTION
**Pull request by jeherve**

### CODE REVIEW

**Comment by lancewillett** [comment]
This looks sane. Could you maybe include before/after screenshots since it's a visual fix?

**Comment by jeherve** [comment]
Before:

screen shot 2016-12-09 at 10 27 14

After:

screen shot 2016-12-09 at 10 28 31

File: admin/jpbeta-admin-main.php [hunk 1]
```diff
@@ -40,6 +40,11 @@
 		list-style-type: none;
 	}
 
+	.j-beta-wrap .card img {
+		max-width: 100%;
+		padding: 1em 1em 1em 0;
```
Comment (lancewillett) [inline admin/jpbeta-admin-main.php]: I think you can simplify this to `padding: 1em 0;`

**Comment by lancewillett** [comment]
Confirmed this fixes the problem. 👍

File: admin/jpbeta-admin-main.php (hunk 1 above)
Comment (lancewillett) [inline admin/jpbeta-admin-main.php]: It worked for me without right padding, in Firefox. Which browser are you testing with?

By setting the width and left padding it should fit in the column nicely.

Comment (jeherve) [inline admin/jpbeta-admin-main.php]: Both work for me indeed.

**Pull request merged**